pkg/apis/config/feature_flags.go

```diff
@@ -138,6 +142,14 @@ func GetFeatureFlagsConfigName() string {
 	return "feature-flags"
 }
 
+func getEnforceNonfalsifiabilityValues() map[string]struct{} {
```

Comment:
this function seems to have added with this very specific signature in order to be able to be passed to setStringFeature which is a function declared inline within NewFeatureFlagsFromMap and only used with getEnforceNonfalsifiabilityValues - would it be clearer to instead combine setStringFeature and this function into one that is specific to the getEnforceNonfalsifiability feature?

it seems like setStringFeature is created with the  idea that it would be reused, however it's declared inline within the NewFeatureFlagsFromMap function so it doesn't have the benefit of being a separate function that could be in its own library with its own tests, and it's not actually used anywhere else, so it seems like it having the indirection of getEnforceNonfalsifiabilityValues + setStringFeature at the moment doesn't give us much gain and makes it a bit harder to read the code

Reply:
I thought we could have a internal function setStringFeature, similar to setFeature above, but as pointed out this is currently being used only for setting the EnforceNonFalsifiablity flag. 
So as suggested, moved all the functionality into one function getEnforceNonfalsifiabilityFeature.